Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:14-alpine
 
+ENV NODE_OPTIONS=--max-old-space-size=4096
```

Comment:
We seem to have this set to 8 GB on CircleCI, do we want to make that update here as well, just to be safe?

`.circleci/config.yml` (Ln 8)
```
NODE_OPTIONS: "--max-old-space-size=8192"
```